src/handwritten/metadata.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
Ensure this is actually bundled in the package. If not, you may need to add it to the [Poetry include config](https://python-poetry.org/docs/pyproject/#include-and-exclude).